exe-unit/src/lib.rs

```diff
@@ -186,8 +178,21 @@ impl<R: Runtime> ExeUnit<R> {
         transfer_service: Addr<TransferService>,
         ctx: ExecCtx,
     ) -> Result<()> {
+        if let ExeScriptCommand::Terminate {} = &ctx.cmd {
+            log::warn!("Terminating running ExeScripts");
+
+            let exclude_batches = vec![ctx.batch_id];
+            let set_state = SetState::default()
+                .state(StatePair(State::Initialized, None))
```

Comment:
is it intentional to set the `Initialized` state as a result of the `Terminate` command?